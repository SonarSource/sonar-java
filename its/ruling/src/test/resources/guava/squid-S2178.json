{
'com.google.guava:guava:src/com/google/common/collect/GeneralRange.java':[
123,
151,
151,
160,
160,
],
'com.google.guava:guava:src/com/google/common/math/DoubleMath.java':[
133,
157,
183,
253,
256,
],
'com.google.guava:guava:src/com/google/common/math/IntMath.java':[
63,
323,
481,
],
'com.google.guava:guava:src/com/google/common/math/LongMath.java':[
62,
389,
389,
510,
522,
550,
565,
],
'com.google.guava:guava:src/com/google/common/primitives/Doubles.java':[
107,
],
'com.google.guava:guava:src/com/google/common/primitives/Floats.java':[
103,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/Monitor.java':[
689,
702,
718,
732,
],
}
