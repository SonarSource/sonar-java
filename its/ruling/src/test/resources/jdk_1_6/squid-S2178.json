{
'jdk_1_6:java/beans/FeatureDescriptor.java':[
212,
213,
214,
],
'jdk_1_6:java/beans/PropertyDescriptor.java':[
549,
550,
],
'jdk_1_6:java/sql/Time.java':[
82,
82,
],
'jdk_1_6:java/sql/Timestamp.java':[
236,
236,
241,
],
'jdk_1_6:java/util/Properties.java':[
618,
],
'jdk_1_6:java/util/regex/Pattern.java':[
3839,
4048,
4134,
4550,
4550,
],
}
