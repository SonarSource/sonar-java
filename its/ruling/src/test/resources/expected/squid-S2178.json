{
'project:com/sun/org/apache/xerces/internal/impl/XMLEntityManager.java':[
1396,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/XSComplexTypeDecl.java':[
495,
500,
510,
],
'project:com/sun/org/apache/xerces/internal/xpointer/ShortHandPointer.java':[
264,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/sax2dtm/SAX2DTM.java':[
1871,
],
'project:com/sun/org/apache/xpath/internal/axes/WalkerFactory.java':[
1563,
],
'project:com/sun/security/auth/module/Krb5LoginModule.java':[
919,
],
'project:java/beans/FeatureDescriptor.java':[
232,
233,
234,
],
'project:java/beans/PropertyDescriptor.java':[
609,
610,
],
'project:java/lang/invoke/AdapterMethodHandle.java':[
661,
679,
],
'project:java/sql/Time.java':[
100,
100,
],
'project:java/sql/Timestamp.java':[
239,
239,
244,
],
'project:java/util/ComparableTimSort.java':[
702,
821,
],
'project:java/util/Properties.java':[
634,
],
'project:java/util/TimSort.java':[
735,
854,
],
'project:java/util/concurrent/ConcurrentLinkedDeque.java':[
503,
512,
],
'project:java/util/regex/Pattern.java':[
4227,
4436,
4522,
4938,
4938,
],
'project:javax/imageio/ImageTypeSpecifier.java':[
1000,
],
'project:javax/swing/JTable.java':[
4965,
4968,
],
}
